algebird-core/src/main/scala/com/twitter/algebird/Field.scala

```diff
@@ -54,6 +54,14 @@ object FloatField extends Field[Float] {
     assertNotZero(r)
     l / r
   }
+  override def sum(t: TraversableOnce[Float]): Float = {
+    var sum = 0.0f
+    t.foreach(sum += _)
```

Comment:
that's a good point, didn't realize that TraversableOnce only has a conversion to Iterator.